CMakeLists.txt

```diff
@@ -209,6 +213,12 @@ if (DOWNLOAD_TITLE_SEQUENCES)
             file(REMOVE \$ENV{DESTDIR}${CMAKE_INSTALL_FULL_DATADIR}/${PROJECT}/title/title-sequences.zip)\n\
         endif ()")
 endif ()
+if (DOWNLOAD_OBJECTS)
```

Comment:
This should allow for pre-caching the objects, like we do with title sequences

Reply:
This should be considered a CI use case, where relevant files are placed in expected locations prior to running `cmake`. See https://github.com/OpenRCT2/OpenRCT2/pull/7195 and https://github.com/OpenRCT2/OpenRCT2/pull/7189